cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -171,11 +171,58 @@ The application may return a different [HTTP Error code](https://developer.mozil
 
 Error disclosure can also be used as a discrepancy factor, consult the [error handling cheat sheet](Error_Handling_Cheat_Sheet.md) regarding the global handling of different errors in an application.
 
-## Prevent Brute-Force Attacks
+## Protect Against Automated Attacks
 
-If an attacker is able to guess passwords without the account becoming disabled due to failed authentication attempts, the attacker has an opportunity to continue with a brute force attack until the account is compromised. Automating brute-force/password guessing attacks on web applications is a trivial challenge. Password lockout mechanisms should be employed that lock out an account if more than a preset number of unsuccessful login attempts are made. Password lockout mechanisms have a logical weakness. An attacker that undertakes a large number of authentication attempts on known account names can produce a result that locks out entire blocks of user accounts. Given that the intent of a password lockout system is to protect from brute-force attacks, a sensible strategy is to lockout accounts for a period of time (e.g., 20 minutes). This significantly slows down attackers, while allowing the accounts to reopen automatically for legitimate users.
+There are a number of different types of automated attacks that attackers can use to try and compromise user accounts. The most common types are listed below:
 
-Also, multi-factor authentication is a very powerful deterrent when trying to prevent brute force attacks since the credentials are a moving target. When multi-factor is implemented and active, account lockout may no longer be necessary.
+| Attack Type | Description |
+|-------------|-------------|
+| Brute Force | Testing multiple passwords from dictionary or other source against a single account. |
+| Credential Stuffing | Testing username/password pairs obtained from the breach of another site. |
+| Password Spraying | Testing a single weak password against a large number of different accounts.|
+
+Various different protection mechanisms can be implemented to protect against them. In many cases these defences do not provide complete protection, but when a number of them are implemented in a defence-in-depth approach, a reasonable level of protection can be achieved.
+
+The follow sections will focus primarily on preventing brute-force attacks, although these controls can also be effective against other types of attacks. For a further guidance on defending against credential stuffing and password spraying, see the [Credential Stuffing Cheat Sheet](Credential_Stuffing_Prevention_Cheat_Sheet.md).
```

Comment:
- `following` instead of `follow`
- What do you think about this? Just looking at what looks/sounds better:
   - against other automated attacks **vs** against other types of attacks
- `For further` instead of `For a further`

Reply:
Some of the defences would also help against non-automated attacks (such as MFA protecting against phishing/etc), so I think the current wording is a bit more general.